cogs/util.py

```diff
@@ -123,6 +124,11 @@ async def remove(self, ctx: commands.Context, *cogs: str):
         else:
             await ctx.send("An argument is required!")
 
```

Comment:
do we make the reboot, shutdown, reload cog commands have owner only permissions too?

Reply:
sure, will do